BookingsAPI/Bookings.AcceptanceTests/Features/SuitabilityAnswers.feature

```diff
@@ -0,0 +1,21 @@
+﻿Feature: SuitabilityAnswer
+	In order to view all suitability answers
+	As an api service
```

Comment:
😄 as an api service

Reply:
Its the same in all the feature files. I just kept ir consistent.

Reply:
well, it's just text 😄